### PR TITLE
Camera photos

### PR DESCRIPTION
**Pull request by ykotturi**
This PR integrates camera and image picker (image upload) native functionalities (see screenshots below).

This PR also removes the outdated client folder

IMG_8657

Clicking "Share a concept" takes you to:

IMG_8658

Clicking "Pick an image from camera roll" prompts user for require permission, and if granted shows:

IMG_8659

Clicking "Take a picture" prompts user for require permission, and if granted shows:

IMG_8660

After image is taken or selected, it is rendered in view and stored locally in app's cache:
IMG_8661

### CODE REVIEW

File: peerdea-native/screens/ShareConceptScreen.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  };
+
+  _takePicture = async () => {
+    await this.askPermissionsAsync();
+    let result = await ImagePicker.launchCameraAsync({
+      allowsEditing: true,
+      aspect: [4, 3],
+      base64: false,
+    });
+    this.setState({ result });
```
Comment (machang19) [inline peerdea-native/screens/ShareConceptScreen.js]: What is the point of this line 66? It seems like we set state in the next two lines

File: peerdea-native/screens/ShareConceptScreen.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          onPress={this._takePicture}
+        />
+        {image &&
+          <Image source={{ uri: image }} style={{ width: 200, height: 200 }} />}
+      </View>
+    );
+  }
+
+  _pickImage = async () => {
+    await this.askPermissionsAsync();
```
Comment (machang19) [inline peerdea-native/screens/ShareConceptScreen.js]: Why do we ask permissions for both camera roll and camera for picking images? It seems like we should just have to ask camera roll permission

File: peerdea-native/screens/ShareConceptScreen.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    console.log('RESULT IS' + result);
+    console.log('RESULT.URI IS' + result.uri);
+
+    if (!result.cancelled) {
+      this.setState({ image: result.uri });
+    }
+  };
+
+  _takePicture = async () => {
+    await this.askPermissionsAsync();
```
Comment (machang19) [inline peerdea-native/screens/ShareConceptScreen.js]: Similar to above, why not just ask for camera here?

File: peerdea-native/screens/ShareConceptScreen.js (hunk 1 above)
Comment (ykotturi) [reply peerdea-native/screens/ShareConceptScreen.js]: Good catch, removed redundancy

File: peerdea-native/screens/ShareConceptScreen.js (hunk 2 above)
Comment (ykotturi) [reply peerdea-native/screens/ShareConceptScreen.js]: From what I've seen documentation wise thus far, the two (image picker and camera) handle different types of permissions, so you need both: https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/requesting_authorization_for_media_capture_on_ios

My personal experience as app user reiterates this, as I've always been prompted to give permission for camera and image upload separately (and access to microphone, which we would need to also prompt for if we were supporting video)

**Comment by machang19** [comment]
+1

**Pull request merged**